### PR TITLE
HBase README updates

### LINKED ISSUE #321: update hbase README to suggest presplitting
In particular, include the shell commands given [on the JIDA referenced by the hbase ref guide](https://issues.apache.org/jira/browse/HBASE-4163)

### PR DESCRIPTION
**Pull request by cmccoy**
Note pre-splitting, document 'columnfamily' parameter.

Fixes #321

### CODE REVIEW

**Comment by busbey** [comment]
Nit: use [hbase] prefix for commit messagr

**Comment by cmccoy** [comment]
Done.

File: hbase/README.md [hunk 1]
```diff
@@ -17,11 +17,13 @@ mvn clean package
...
 ### 3. Create a HBase table for testing
 
-```
-/HBASE-HOME-DIR/bin/hbase shell
+For best results, use the pre-splitting strategy recommended in [HBASE-4163](https://issues.apache.org/jira/browse/HBASE-4163):
 
-hbase(main):001:0> create 'usertable', 'family'
 ```
+hbase(main):001:0> create 'usertable', 'family', {SPLITS => (1..200).map {|i| "user#{1000+i*(9999-1000)/200}"}, MAX_FILESIZE => 4*1024**3}
+```
```
Comment (busbey) [inline hbase/README.md]: The max filesize bit is questionable.

Should note that this makes 200 regions, and you should adjust dependg on cluster size. Should be a sizinh guide for regions per server in the hbase ref guide.

Comment (cmccoy) [inline hbase/README.md]: > the max filesize bit is questionable

I just grabbed the recommendations from the bug. Would you prefer to drop that? The important piece is to split on user####### boundaries...

**Comment by busbey** [comment]
+1

**Pull request merged**